source/common/common/logger.cc

```diff
@@ -14,9 +14,11 @@ namespace Logger {
 
 #define GENERATE_LOGGER(X) Logger(#X),
 
+std::string Logger::default_log_format = "[%Y-%m-%d %T.%e][%t][%l][%n] %v";
```

Comment:
technically, this runs afoul of the static initializer rule. I would probably just implement as a static function using CONSTRUCT_ON_FIRST_USE or something like that.

Reply:
Could we not just make it a const char array instead of a string?

Reply:
Sure that seems better.